server/api/endpoints/cart.js

```diff
@@ -0,0 +1,47 @@
+import { Cart } from "../../../lib/collections";
+
+export function cart(Api) {
+  Api.addCollection(Cart);
+
+  Api.addRoute("cart", { authRequired: false }, {
+    get: () => {
+      return Cart.find().fetch();
+    },
+    post: {
+      action: () => {
+        if (Cart.insert(this.request.body)) {
+          return {
+            status: "success",
+            statusCode: 200,
```

Comment:
Replace the hardcoded http-status codes with this https://www.npmjs.com/package/http-status